### DISCUSSION TITLE
Passing PHP variables into my Livewire action parameter crashes my action method

### DISCUSSION THREAD

**Discussion by TheSearcher** [livewire/livewire#5389] [1. Help]
Passing PHP variables into my Livewire action parameter crashes it.
```

 @php {{ $tel = 'telephone'; }} @endphp

<button wire:click="sortBy({{$tel}})">
    Add Todo
</button>
```

This is the error message in my console:

```
VM527:8 Uncaught ReferenceError: telephone is not defined
    at eval (eval at value (wire-directives.js:93:24), <anonymous>:8:16)
    at Directive.value (wire-directives.js:100:22)
    at Directive.get (wire-directives.js:58:33)
    at node_initializer.js:209:42
    at Component.value (index.js:582:9)
    at HTMLButtonElement.<anonymous> (node_initializer.js:201:23)

```
When I pass a simple string into the same method it works i.e

```
<button wire:click="sortBy('telephone')">
    Add Todo
</button>
```

**Comment by joshhanley** [comment]
@TheSearcher the issue is that you are echoing out a string (by using `{{ $tel }}`) anything that is inside the `wire:click` is a javascript expression, so it's looking for a variable called `telephone`. To fix this, you need to put single quotes around the outside of the blade echo, but inside the parenthesis, like this: `wire:click="sortBy('{{ $tel }}')"`.

Hope this helps!
